.github/workflows/automerge_scan.yml

```diff
@@ -0,0 +1,58 @@
+name: AutoMergeScan
+on:
+  schedule:
+    - cron: '31 */2 * * *'
+  workflow_dispatch:
+
+jobs:
+  automerge_scan:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Debug
+      env:
+        TOKEN: ${{ secrets.TOKEN }}
+      run: |
+        set -e
+
+        echo ${TOKEN} | gh auth login --with-token
+        gh pr list -R sonic-net/sonic-buildimage -A mssonicbld --json additions,assignees,author,baseRefName,body,changedFiles,closed,closedAt,comments,commits,createdAt,deletions,files,headRefName,headRepository,headRepositoryOwner,id,isCrossRepository,isDraft,labels,latestReviews,maintainerCanModify,mergeCommit,mergeStateStatus,mergeable,mergedAt,mergedBy,milestone,number,potentialMergeCommit,projectCards,reactionGroups,reviewDecision,reviewRequests,reviews,state,statusCheckRollup,title,updatedAt,url > prs.log
+        cat prs.log | jq
+    - name: Main
+      run: |
+        set -e
+
+        count=$(cat prs.log | jq 'length')
+        for ((i=0;i<$count;i++))
+        do
+          url=$(cat prs.log | jq -r ".[$i].url")
+          echo PR: $(($i+1))/$count, URL: $url
+          [[ "$url" == "" ]] && continue
+          checks=$(cat prs.log | jq ".[$i].statusCheckRollup")
+          checks_count=$(echo $checks | jq 'length')
```

Comment:
The checks_count should be greater than 1, right?
Can we check the required checks?

Reply:
It is easier to check all checks instead of required checks.